### PR TITLE
Add ERC-4926: Composability Extension For ERC-721 Standard

### PR DESCRIPTION
**Pull request by muhammedea**
An extension of the ERC-721 standard to enable NFT tokens to own other NFT tokens.

### CODE REVIEW

**Review by DELJUJAZI — APPROVED** [review]

File: EIPS/eip-4911.md [hunk 1]
```diff
@@ -0,0 +1,157 @@
+---
+eip: 4911
```
Comment (SamWilsn) [inline EIPS/eip-4911.md:2]:
```suggestion
eip: xxxx
```
An EIP number will be assigned when you move this PR out of draft. Unfortunately you can't pick a specific EIP number.

File: EIPS/eip-4911.md [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+category: ERC
+created: 2022-02-15
+requires: 165, 721
+---
+
+# Composability Extension For ERC-721 Standard
+
+## Abstract
+
+This standard outlines a smart contract interface that should be applied to ERC721 contracts to add composability feature.
```
Comment (SamWilsn) [inline EIPS/eip-4911.md:18]:
```suggestion
This standard outlines a smart contract interface that should be applied to [EIP-721](./EIP-721.md) contracts to add composability feature.
```

See [EIP-1 § Linking to other EIPs](https://eips.ethereum.org/EIPS/eip-1#linking-to-other-eips)

File: EIPS/eip-4911.md [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+author: Muhammed Emin Aydın (@muhammedea), Dmitry Savonin (@dmitry123)
+discussions-to: https://ethereum-magicians.org/t/erc-4911-composability-extension-for-erc-721-standard/8304
+status: Draft
+type: Standards Track
+category: ERC
+created: 2022-02-15
+requires: 165, 721
+---
+
+# Composability Extension For ERC-721 Standard
```
Comment (SamWilsn) [inline EIPS/eip-4911.md:14]:
```suggestion
```

The title will get added for you when the EIP is rendered.

File: EIPS/eip-4911.md [hunk 4]
```diff
@@ -0,0 +1,157 @@
...
+        @dev This function gives child token ids for a given parent id and slot.
+        @param slotId : slot id that the child tokens are attached
+        @param parentId : the parent token id
+    */
+    function getChildTokens(uint256 slotId, uint256 parentId) external returns(uint256[]);
+}
+
+
+```
+
```
Comment (SamWilsn) [inline EIPS/eip-4911.md:147]: You'll need to add the `Rationale` section.

**Review by SamWilsn — CHANGES_REQUESTED** [review]

**Review by ed738 — APPROVED** [review]

File: EIPS/eip-4911.md [hunk 5]
```diff
@@ -0,0 +1,157 @@
...
+## Security Considerations
+In this standard, every child token will be stored on the parent contract. So parent contract should be ERC721Receiver.
+
+Attach actions requires approval from child token contracts to the parent contract.
+
+Given checks for every function should be implemented.
+
+
+## Copyright
+Copyright and related rights waived via [CC0](https://creativecommons.org/publicdomain/zero/1.0/).
```
Comment (Pandapip1) [inline EIPS/eip-4911.md:157]:
```suggestion
Copyright and related rights waived via [CC0](../LICENSE).
```

**Comment by Pandapip1** [comment]
@eth-bot rerun

**Comment by Pandapip1** [comment]
Closing since the author hasn't been seen in >3 months (the stale threshold).

**Pull request closed without merging**